Makefile

```diff
@@ -11,6 +11,18 @@ DB_SERVER_URL = 'jdbc:postgresql://postgresdbserver:5432/sysml2'
 DB_USER = 'postgres'
 DB_PASSWORD = 'mysecretpassword'
 
+# URL for the docker repository to push the image to.
+DOCKER_REPOSITORY=registry-1.docker.io
```

Comment:
Is it really best practice to point to this particular hostname?  I dug around a bit and it wasn't clear to me the whys and hows of such.

Reply:
I'm actually not sure. I'll try removing it and see what happens; it's basically a copy of something done in another Docker related makefile we have, which I did not write originally.

Reply:
It works fine as just `docker.io` rather than a specific mirror, so we'll go with that.